src/main/java/pl/wedding/management/model/reservation/Reservation.java

```diff
@@ -0,0 +1,29 @@
+package pl.wedding.management.model.reservation;
```

Comment:
Złe katalogowanie klasy. Mogłeś przeoczyć informację, że będziemy tworzyli aplikacje w architekturze modularnej. To taka uproszczona wersja architektury hexagonalnej z wystawioną fasadą. Odsyłam do filmiku, który jest podpięty na Slacku. :)